Source/EasyTipView.swift

```diff
@@ -139,7 +139,6 @@ public extension EasyTipView {
             self.alpha = self.preferences.animating.dismissFinalAlpha
         }) { (finished) -> Void in
             completion?()
-            self.delegate?.easyTipViewDidDismiss(self)
```

Comment:
thanks for the pr, but the fix is actually on line 349, calling `dismiss()` instead of what is there at the moment.. if you make that change I won't push the change myself and I'll merge your pr